spec/features/reading_pages_spec.rb

```diff
@@ -159,6 +159,7 @@ class Check < Struct.new(:path, :source_link, :fragment, keyword_init: true)
       /docs/tutorials/sso-setup-with-graphql
       /docs/webhooks/setup
       /docs/webhooks
+      /docs/pipelines/my-builds
```

Comment:
I'm all for speccing this — yes, please! — but I'm not sure this is the right spot for this. 🤔 This spec is about making sure that redirects work — that old URLs go to current URLs. This isn't a redirect, nor an 'old' url, it's a 'new' url which hasn't been published anywhere yet.

I did consider adding a new spec which iterates all `Rails.root.join("pages").glob("**/*.md{,.erb}")` paths and checks that the matching urls render, but it seems like it had a lot of overlap with the spidery spec, and I couldn't figure out a nice way to do that — plus there were some pages which were broken and I was out of decision making power about what to do with them.

Reply:
This is the spec I wrote:
https://github.com/buildkite/docs/compare/spec-all-pages

but it probably needs to be merged with the `clicking links` spec.

Reply:
I'll defer to @sj26 on the actual implementation of speccing this. 

But as an alternative suggestion, what if we listed all unlisted pages :-p. That way unless you intentionally add the page to an exclusion, a page without a link errors, which is probably (?) a more useful check?

Reply:
a4d5e2c spec tests 139 paths which 69 of them are covered in `clicking links`. So we are testing 70 extra paths, which I reckon is a win and how to properly marry them, we can think about later... 😅

Reply:
@JuanitoFatas yeah, good call, and nice work! Sorry, I ran out of energy to dig deeper last night.